repo_structure/repolint.json

```
@@ -6,8 +6,8 @@
     },
     "rules": {
       "all": {
-        "license-file-matches:file-hash": ["error", {"file": "LICENSE*", "hash":"b40930bbcf80744c86c46a12bc9da056641d722716c378f5659b9e555ef833e1"}],
-        "code-of-conduct-file-matches:file-hash": ["error", {"file": "CODE_OF_CONDUCT.md", "hash":"d2719e3ce1c383567a5f1bfe8f5c5af2042b3e15b5620b8d8a4b5ada022289e4"}],
+        "apache-license-file:file-contents": ["error", {"files": ["LICENSE*"], "content": "Apache License.*Version 2.0", "fail-on-non-existent": true}],
+        "code-of-conduct-file:file-contents": ["error", {"files": ["CODE_OF_CONDUCT*"], "content": "https://wiki.hyperledger.org/community/hyperledger-project-code-of-conduct", "fail-on-non-existent": true}],
         "security-file-matches:file-hash": ["error", {"file": "SECURITY.md", "hash":"68457718b668ca838596f4a0e47359ccc0f34fb7695605bc46c3248dfffc65ab"}],
```

Did you mean to leave security out?  I think the Wiki URL for the defect response would be a good "magic word"
```suggestion
        "security-file-matches:file-contents": ["error", {"files": ["SECURITY.md"], "content":"https://wiki.hyperledger.org/display/SEC/Defect+Response", "fail-on-non-existent": true}],
 ```

---

You're right, we might as well cover this file the same way. This doesn't work as you suggest though because the plus sign is interpreted as a special regexp character and it turns out that simply escaping it seems to trigger a bug in repolinter so it needs a double escape! So, maybe just using a dot instead is good enough. I'll update the PR. Thanks.